### ISSUE TITLE
Replacing what's happening with trends

### ISSUE THREAD

**Issue by zetorplugger** [Bl4Cc4t/GoodTwitter2#154]
I don't know if this is possible but twitter made yet another utterly useless change
and the trends no longer visible on the side bar but instead what's happening is

the trends are now only visible when you click show more and then Trending

Is there a way to load the data on https://twitter.com/explore/tabs/trending
into the sidebar as it used to be

![whatshappening](https://user-images.githubusercontent.com/66390092/86638164-1b975d80-bfd7-11ea-8f66-831e5126493f.JPG)
![trend](https://user-images.githubusercontent.com/66390092/86638178-22be6b80-bfd7-11ea-92be-234175b287db.JPG)

**Comment by Bl4Cc4t** [comment]
I think this change is country specific atm, I can not access the site you linked.
So this has to wait until it reaches Germany unfortunately (I don’t think it will take long).

Funfact: [i/timeline](https://twitter.com/i/timeline) looks like this
![image](https://user-images.githubusercontent.com/6740726/87224523-3de9fc00-c386-11ea-8e91-301496a518c9.png)

Seems like nothing is happening at the moment.

**Comment by zetorplugger** [comment]
I am missing something? 
No matter what IP I use What's happening appears where trends appeared before so it's not IP based.
Is there any setting on twitter itself that can change which country twitter believes I am in?
This is how it looks  without using your scripts
![happening](https://user-images.githubusercontent.com/66390092/87228015-bc529800-c39e-11ea-8701-cfa1a6a8829d.JPG)

**Comment by Bl4Cc4t** [comment]
Normally you can change the trends settings by clicking the cog on the top right.
![image](https://user-images.githubusercontent.com/6740726/87247750-ed41d400-c455-11ea-94ea-320a9719b8e6.png)
But now I noticed you don’t even have that on your screenshot...?
Can you visit [settings/trends](https://twitter.com/settings/trends)?

**Comment by zetorplugger** [comment]
> Can you visit [settings/trends](https://twitter.com/settings/trends)?

Yes I can by the problem is no matter what country I set there I don't get the trend list but 
What's happening list

**Comment by LeoKesler** [comment]
I have exactly the same issue. And I am in Brazil.
Same happens in Firefox Nightly Win10, Firefox Nightly Linux, Opera Linux.

**Comment by Bl4Cc4t** [comment]
Sorry, can’t do much about that until this change reaches my country.
